benchmark_runner/workloads/workloads_operations.py

```diff
@@ -48,6 +49,12 @@ def __init__(self):
         self._es_user = self._environment_variables_dict.get('elasticsearch_user', '')
         self._es_password = self._environment_variables_dict.get('elasticsearch_password', '')
         self._es_url_protocol = self._environment_variables_dict['elasticsearch_url_protocol']
+        self._scale = self._environment_variables_dict.get('scale', '')
```

Comment:
Really need to find a way to not embed knowledge of particular workloads in this file.

Reply:
Its not related to particular workload vdbench pod or vm, it will also support other workloads.

Reply:
Yep.